sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -5,14 +5,12 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
     <Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-    <Version>4.6.0</Version>
+    <Version>4.7.0</Version>
     <AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
     <PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
     <PackageReleaseNotes>
   <![CDATA[
-- Added SAP HANA partition support in Copy Activity
-- Added property prefix for azure blob source
-- Added support for writing data into SFTP server using Copy activity
+- Added compute support in execute data flow activity
```

Comment:
Hi Shawn @shawnxzq , which PR contains this change? I just want to double check it's correct to update to version 4.7.0

Reply:
Here it is: https://github.com/shawnxzq/azure-sdk-for-net/commit/7c5ce9be0967c8ae710260164eb85c3c30986a23#diff-b2fed88956b4c203bef972002228c4d9